gnd/src/main/java/com/google/android/gnd/ui/home/HomeScreenViewModel.java

```diff
@@ -73,6 +73,8 @@
 
   private final Subject<Nil> savePolygonRequest = PublishSubject.create();
 
+  private final Subject<Nil> undoPolygonPoints = PublishSubject.create();
```

Comment:
`removeLastVertex`

This should live in `MapContainerViewModel`, or better, a viewmodel specific to the polygon drawing mode, not the home model.

Reply:
we already discussed this.